src/main/resources/migration/common/V57__add_next_review_date.sql

```diff
@@ -0,0 +1,51 @@
+alter table plan
+    add column if not exists next_review_date date,
+    add column if not exists closed_date      date;
+
+-- index for review date required for determining late reviews
+create index if not exists idx_plan_next_review_date on plan (next_review_date);
+
+comment on column plan.next_review_date is 'Next review date is calculated and stored in this field for performance reasons. This will be the first case review date when no reviews exist or the next review date of the most recent review when a review exists.';
+comment on column plan.closed_date is 'Closed date is calculated and stored in this field for performance reasons. This will be the closed date from the review that closes the CSIP.';
+
+update plan
+set next_review_date = case
+                           when exists(select 1 from review where review.plan_id = plan.plan_id)
+                               then (select next_review_date
+                                     from review
+                                     where review.plan_id = plan.plan_id
+                                     order by review.review_sequence desc
+                                     limit 1)
+                           else plan.first_case_review_date end
+where plan.next_review_date is null;
+
```

Comment:
I can't see where the closed_date for existing plans is populated by this script